Examples/Movies/MovieCell.js

```diff
@@ -22,7 +22,7 @@ var {
   StyleSheet,
   Text,
   TouchableHighlight,
-  View
+  View,
```

Comment:
We've had on our todolist to draft a fb coding style guide for a while. Would be nice if you want to do it and follow what we do on react/react native codebase :)

@sebmck we want to always have trailing comma everywhere